spec/movie_spec.rb

```diff
@@ -0,0 +1,19 @@
+require_relative '../classes/movie'
+
+describe Movie do
+  before :each do
+    @movie = Movie.new('loovee', '2021-03-10', true)
+  end
+
+  context 'When getting the name of a movie' do
+    it 'should return the correct movie name' do
+      expect(@movie.name).to eq('loovee')
+    end
+  end
+
+  context 'When checking the published date' do
+    it 'it should return a published date' do
+      expect(@movie.publish_date).to eq('2021-03-10')
+    end
+  end
+end
```

Comment:
Tests in this file are also failing. Kindly fix these errors.
![image](https://user-images.githubusercontent.com/33268927/193262010-702c2763-72bc-4721-9a60-559dd3b35c9e.png)